_posts/2016/2016-03-16-team-orange-mallorca.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "A team of ePagees conquers Tierra Buena in Mallorca"
+date: "2016-03-16 06:16:03"
+icon: suitcase
+categories: working-at-epages
+authors: ["Alejandra A., Benjamin N. and Heiko A."]
+---
+
+Last week Team Orange, the ePages release and test automation team, was welcomed by the precious views, hearty people and refreshing atmosphere of Mallorca. The purpose of this particular trip was to celebrate an off-site team event liberated from the daily tasks and routines of the business life. The team set off to new shores to develop a master plan on how it could face the winds of change of the advancement of each team member. The ultimate goal was to find the best way for contributing to the promising future of the whole research and development department.
+
+Now we are back and ready to briefly tell you the tale of the fearless developers' journey to a mysterious island. So please enjoy the ambigous report!
```

Comment:
So please enjoy the ambigous report! > Enjoy this ambiguous report.

What exactly would you like to express with "ambiguous"? Might sound a bit "ironic"?